### DISCUSSION TITLE
install on fire TV stick ?

### DISCUSSION THREAD

**Discussion by rajdude** [mvysny/photocloud-frame-slideshow#182] [Q&A]
Is it possible to install this app on Fire TV stick? Like sideload using an APK file?

I really need to be able to view all my RAW image files (.dng) kept on my local networked Windows file server on my TV. My ecosystem has a few Fire TV sticks around the house. 

I am able to view all my raw files using this software on my Android phones and tablets just fine.

**Comment by mvysny** [comment]
Hi, the official PhotoCloud release requires Google Play to be present on the Android device. If I understand correctly, Fire TV lacks Google Play and thus the official PhotoCloud may not work.

However, there are "unofficial" releases of PhotoCloud which should work on any Android, given that it's at least Android 4.1 (ideally Android 5). If you're interested, just let me know and I'll post a link to the APK.

**Comment by rajdude** [comment]
Hi Martin, yes! Please post a link to the APK 🙂

-Raj
Sent from my cell

Get BlueMail for Android<https://bluemail.me>
On Feb 27, 2023, at 5:59 AM, Martin Vysny ***@***.******@***.***>> wrote:

Hi, the official PhotoCloud release requires Google Play to be present on the Android device. If I understand correctly, Fire TV lacks Google Play and thus the official PhotoCloud may not work.

However, there are "unofficial" releases of PhotoCloud which should work on any Android, given that it's at least Android 4.1 (ideally Android 5). If you're interested, just let me know and I'll post a link to the APK.

**Reply by mvysny** [reply]
Please try the following folder: https://drive.google.com/file/d/1wPxojUkYPpVo4CRLnK3qaP9n7LQaBc1E/view?usp=share_link

It contains a bunch of APKs which are just different versions of the same PhotoCloud. Please try the newest one, and if it doesn't work, please try the older one instead.

**Comment by rajdude** [comment]
Thanks Martin,
That link is taking me to a screenshot, no files there. Maybe I am doing something wrong?

[cid:3598bc36-4d16-4775-9e0f-43b2ab5bd514]



Sincerely,

Rajeev Mehra

My website - rajdude.com<http://rajdude.com>


________________________________

**Reply by mvysny** [reply]
Sorry, I've posted an incorrect link by accident. This is the correct one: https://drive.google.com/drive/folders/145so1MQaABb8EwaowLGrisX9mDUqOj6J
